fs/layer/node.go

```diff
@@ -112,6 +113,53 @@ var fuseOpFailureMetrics = map[string]string{
 	fuseOpWhiteoutGetattr: commonmetrics.FuseWhiteoutGetattrFailureCount,
 }
 
+// FuseOperationCounter collects number of invocations of the various FUSE implementations and emits them as metrics.
+// Setting `waitPeriod` to be > 0 allows delaying the time when the metrics are emitted.
+type FuseOperationCounter struct {
+	opCounts    map[string]*atomic.Int32
+	waitPeriod  time.Duration
+	imageDigest digest.Digest
+}
+
+var FuseMetricsList = []string{
```

Comment:
`FuseMetricsList` -> `FuseOpList`. Also can you add a comment here or change to not export.